sql/2021/css/box_sizing.sql

```diff
@@ -0,0 +1,39 @@
+#standardSQL
+# 1. Distribution of the number of occurrences of box-sizing:border-box per page.
+# 2. Percent of pages with that style.
+CREATE TEMPORARY FUNCTION countBorderBoxDeclarations(css STRING)
+RETURNS NUMERIC
+LANGUAGE js
+OPTIONS (library = "gs://httparchive/lib/css-utils.js")
+AS '''
+try {
+  const ast = JSON.parse(css);
+  return countDeclarations(ast.stylesheet.rules, {properties: /^(-(o|moz|webkit|ms)-)?box-sizing$/, values: 'border-box'});
+} catch (e) {
+  return null;
+}
+''';
+
+SELECT
+  percentile,
+  client,
+  COUNT(DISTINCT IF(declarations > 0, page, NULL)) AS pages,
+  COUNT(DISTINCT page) AS total,
+  COUNT(DISTINCT IF(declarations > 0, page, NULL)) / COUNT(DISTINCT page) AS pct_pages,
+  APPROX_QUANTILES(declarations, 1000 IGNORE NULLS)[OFFSET(percentile * 10)] AS declarations_per_page
+FROM (
+  SELECT
+    client,
+    page,
+    countBorderBoxDeclarations(css) AS declarations
+  FROM
+    `httparchive.almanac.parsed_css`
+  WHERE
+    date = '2021-07-01'),
+  UNNEST([10, 25, 50, 75, 90]) AS percentile
```

Comment:
Potential issue with UNNEST and count.

Reply:
This should be ok given the use of `DISTINCT page`

Reply:
Agreed on a second look. Happy to resolve.